### ISSUE TITLE
direct lyric copy from screen (translated lyrics)

### ISSUE THREAD

**Issue by Learner00100** [fast4x/RiMusic#1003]
### Description

.

### Suggested Solution

option to copy lyrics from screen itself, like long press the lyric will enable selction option so that part of lyric of our choice can be copied.
this is helpful to copy translated lyrics from screen.
implement option in unsync lyric alone, because selecting  the lyric of sync lyric is mess up because it will play the song of that timestamp.

**Comment by fast4x** [comment]
https://github.com/fast4x/RiMusic/assets/1484476/4fe2a801-a959-499d-9949-62f81afa23b8

But when select text, system copy options is showed... Long click in RiMusic would be a duplicate useless option.

**Comment by Learner00100** [comment]
> Screenrecorder-2024-03-10-09-29-17-137.mp4 
> But when select text, system copy options is showed... Long click in RiMusic would be a duplicate useless option.

Is it possible to include translated lyrics underneath the orginal lyrics.
So that we can copy the translated lyrics from that tab

**Comment by dAtA-TRoN** [comment]
By the way, while we're at it: When I press the translation button, the app doesn't translate the text; instead, it adds "/r" everywhere in the text. I have no idea what that's about. Is it a bug or a feature?

![ZomboDroid_10032024110634.jpg](https://github.com/fast4x/RiMusic/assets/71184627/1e2d751f-dc0d-48c8-ac11-e73ed2bc18a7)

**Comment by fast4x** [comment]
> By the way, while we're at it: When I press the translation button, the app doesn't translate the text; instead, it adds "/r" everywhere in the text. I have no idea what that's about. Is it a bug or a feature?
> 
> ![ZomboDroid_10032024110634.jpg](https://github.com/fast4x/RiMusic/assets/71184627/1e2d751f-dc0d-48c8-ac11-e73ed2bc18a7)
> 
> 

What language in app is set?

**Comment by dAtA-TRoN** [comment]
> What language in app is set?

German

**Comment by fast4x** [comment]
uhm it's a mistery, occur always?

**Comment by dAtA-TRoN** [comment]
Let me check

**Comment by dAtA-TRoN** [comment]
> uhm it's a mistery, occur always?

It happens more often than it doesn't happen.

**Comment by dAtA-TRoN** [comment]
> You use firewall?

No, I don't. But let me examine this a little longer so I can give you a more qualitative answer.

**Comment by fast4x** [comment]
Lyrics copy it's just implemented in app look better please...
![Screenshot_20240310_223043](https://github.com/fast4x/RiMusic/assets/1484476/9f337e89-5b8f-4ce1-a504-4774b454abf6)

**Comment by Learner00100** [comment]
> Lyrics copy it's just implemented in app look better please... ![Screenshot_20240310_223043](https://private-user-images.githubusercontent.com/1484476/311545446-9f337e89-5b8f-4ce1-a504-4774b454abf6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMjEyODYsIm5iZiI6MTcxMDEyMDk4NiwicGF0aCI6Ii8xNDg0NDc2LzMxMTU0NTQ0Ni05ZjMzN2U4OS01YjhmLTRjZTEtYTUwNC00Nzc0YjQ1NGFiZjYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTFUMDEzNjI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWJkM2Y0ZTNhMTlhMWFmMGMzYzQ5YjRlM2YxYzA4N2YwNTU5ZjcyYjBjZTZhMmMxNmQxMTViZDIwZTUwMDI2ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.jFpfTvLCGqXTBolPMaEAbi3x2Up9_Q06_wwOy-hnFq0)

in lyrics copy option, Is it possible to include translated lyrics underneath the orginal lyrics.
So that we can copy the translated lyrics from that tab

**Comment by fast4x** [comment]
You are right, translate lyrics can't copied.
 I'll add a fix for that

**Comment by Learner00100** [comment]
> You are right, translate lyrics can't copied. I'll add a fix for that

Thanks for consideration 😃

**Comment by fast4x** [comment]
You are welcome, thank you for reporting.

**Comment by dAtA-TRoN** [comment]
> You are right, translate lyrics can't copied.\nI'll add a fix for that

Perhaps you could also think about relocating the translation button to the menu, rather than keeping it down there. I don't know about you, but having the button at the bottom feels a bit out of place and detracts from the clean look.

![ZomboDroid_11032024033454.jpg](https://github.com/fast4x/RiMusic/assets/71184627/c1692be9-41a1-45ea-826b-5bcb5a074f83)

**Comment by Learner00100** [comment]
> > You are right, translate lyrics can't copied.\nI'll add a fix for that
> 
> Perhaps you could also think about relocating the translation button to the menu, rather than keeping it down there. I don't know about you, but having the button at the bottom feels a bit out of place and detracts from the clean look.
> 
> ![ZomboDroid_11032024033454.jpg](https://private-user-images.githubusercontent.com/71184627/311571240-c1692be9-41a1-45ea-826b-5bcb5a074f83.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxNjk3ODYsIm5iZiI6MTcxMDE2OTQ4NiwicGF0aCI6Ii83MTE4NDYyNy8zMTE1NzEyNDAtYzE2OTJiZTktNDFhMS00NWVhLTgyNmItNWJjYjVhMDc0ZjgzLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzExVDE1MDQ0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJmMjdmYWY4NGJhMGFmZDFlNWJmMDQ3YmU2OTViMjhmYTIzZTA1ZjFkMjUzMDJhNGM5NzU2ZTViNmNkNjhkMDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.X89d_CPXepY_U_vArYe3Fg37oLDLhaZeZpHymlrwUHQ)

But icon their is  better shortcut to get translate.

**Comment by dAtA-TRoN** [comment]
> But icon their is better shortcut to get translate.

It's just one tab away then.

**Comment by fast4x** [comment]
Implemented copy translated lyrics, available from next version
